### PR TITLE
Initial Commit

### PR DESCRIPTION
**Pull request by Iankwok**

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+  border-right: none;
+}
+td[data-num='3'],td[data-num='6']{
+  border-right: 3px solid black;
+
+}
+#row-3, #row-6 {
+  border-bottom: 3px solid black;
+}
+
```
Comment (f3r) [inline css/style.css:46]: please add this to your sublime preferences:

```
    // Set to false to disable scrolling past the end of the buffer.
    // On OS X, this value is overridden in the platform specific settings, so
    // you'll need to place this line in your user settings to override it.
    "scroll_past_end": true,
```

and delete the extra spaces

File: index.html [hunk 2]
```diff
@@ -0,0 +1,127 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <title>Sudoku</title>
+  <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.5/css/bootstrap.min.css">
+  <link rel="stylesheet" href="./css/style.css">
+  <script src="http://ajax.googleapis.com/ajax/libs/jquery/2.1.1/jquery.min.js"></script>
```
Comment (f3r) [inline index.html]: `<script>` should be added just before `</body>`

File: index.html [hunk 3]
```diff
@@ -0,0 +1,182 @@
...
+        <h2>Choose Game Difficulty to Begin Game:<h2>
+        <button type="button" class="btn btn-primary btn-sm" id="easy">Easy</button>
+        <button type="button" class="btn btn-warning btn-md" id="medium">Medium</button>
+        <button type="button" class="btn btn-danger btn-lg" id="hard">Hard</button>
+     </section>
+    </div>
+    <div id="menu1" class="tab-pane fade in">
+    <section class = "container well">
+        <table>
+          <thead>
```
Comment (dshcheung) [inline index.html:40]: fix indentations please

File: index.html [hunk 4]
```diff
@@ -0,0 +1,182 @@
...
+        <div class="modal-header">
+          <h2 class="modal-title text-success"> Well done! You have completed the game in <span id="timeWin">0</span> seconds</h2>
+        </div>
+         <div class="modal-body">
+          <img src="pikachu.png" width="200" height="200" align="middle">
+        </div>
+      </div>
+    </div>
+  </div>
+    </section>
```
Comment (dshcheung) [inline index.html]: fix indentations here too

File: index.html [hunk 5]
```diff
@@ -0,0 +1,182 @@
+  <!DOCTYPE html>
```
Comment (f3r) [inline index.html]: please fix indentation and ping me in slack when done

Comment (f3r) [inline index.html]: https://gist.github.com/f3r/acaa2a3b697deefd51df

**Comment by f3r** [comment]
you forgot to update your `.beautifyrc` file... now have 4 spaces.... please fix again

**Pull request merged**